docs/core/docker/build-container.md

```diff
@@ -146,46 +135,48 @@ From the working folder, enter the *app* folder with the example source code and
 dotnet publish -c Release
 ```
 
-This command compiles your app to the *publish* folder. The path to the *publish* folder from the working folder should be `.\app\bin\Release\netcoreapp2.2\publish\`
+This command compiles your app to the *publish* folder. The path to the *publish* folder from the working folder should be `.\app\bin\Release\netcoreapp3.1\publish\`
 
 Get a directory listing of the publish folder to verify that the *myapp.dll* was created. From the *app* folder, run one of the following commands:
```

Comment:
The instructions don't tell you how to choose which command.  Until now there hasn't been any mention of alternative commands.

Reply:
I don't understand what you mean.

Reply:
Apparently you choose the bash box if you're on LInux, and the console box if you're on Windows? This is never explained.